src/treetensornetworks/opsum_to_ttn.jl

```diff
@@ -57,7 +57,7 @@ function svdTTN(
   cutoff::Float64 = 1e-15,  ###FIXME: better to use eps(coefficient_type) * 1e1 or something similar
 )::TTN where {VT}
   
-
+  edgetype_sites = edgetype(sites)
```

Comment:
Seems like this is needed here?

Reply:
I think we don't have to concern ourselves with updating the style of `svdTTN` since we plan to remove it anyway.

Reply:
Yes, it's not covered by the test anyway, probably should have excluded it from the changes.
Can we simply rename the pre-PR version of `opsum_to_ttn.jl` to `deprecated_opsum_to_ttn.jl` for now?

Reply:
Sure, that's a good idea, and then remove anything from `opsum_to_ttn.jl` unrelated to this PR?